db/migrate/20220523204529_create_entities.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEntities < ActiveRecord::Migration[7.0]
+  def change
+    create_table :entities do |t|
+      t.string :name
+      t.integer :amount
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly fix the column name in all the user's foreign keys, we should reference the author, according to the ERD, we should have an `author_id` as a user's foreign key column name. 

![schema](https://user-images.githubusercontent.com/53124548/170355704-d9341501-fcfb-4c7b-addd-1ee084b2387a.png)